datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -134,12 +130,12 @@ func (f *ChainDataFetcher) Stop() error {
 	return nil
 }
 
-func (f *ChainDataFetcher) makeRequests(start, end uint64, reqType requestType, stopCh chan struct{}) {
+func (f *ChainDataFetcher) sendRequests(start, end uint64, reqType requestType, shouldUpdateCheckpoint bool, stopCh chan struct{}) {
 	for i := start; i <= end; i++ {
 		select {
 		case <-stopCh:
 			return
```

Comment:
It would be better to log to indicate that the sendRequests successfully receives `stopCh`

Reply:
Not necessarily need to be done in this PR.

Reply:
Right. I will add some logs on the next PR.